README.adoc

```diff
@@ -68,41 +62,42 @@ NOTE: This file is where we alias the `latest` path in URLs.
 # Redirect latest imdg alias to the latest version
 /imdg/latest/*  /imdg/4.1.2/:splat 200!
 ----
-
-The documentation site will now be built, including content from your tagged version.
++
+- Update the `search-config.json` file with your release version of IMDG.
```

Comment:
The only change in the `search-config.json` is to change content of `variables.version` to released version, right? 
For example change `"version": ["4.2"]` to `"version": ["4.3"]` if `4.3` is released. Nothing is added, just changed.